src/main/kotlin/com/zepben/evolve/services/network/tracing/feeder/AssignToLvFeeders.kt

```diff
@@ -96,17 +95,9 @@ class AssignToLvFeeders {
                 terminal,
                 { eq, feeder ->
                     eq.addContainer(feeder)
-                    // Handle classes extending Equipment
-                    when (eq) {
-                        is ProtectedSwitch -> eq.operatedByProtectionEquipment.forEach { pe -> pe.addContainer(feeder) }
-                    }
```

Comment:
As `ProtectionRelayFunction` is a `PowerSystemResource` but not an `Equipment`, this cannot be immediately translated. Question: What about the function's sensors and the system of each of its schemes? Should they be included in the container?

Reply:
Just the ProtectionRelaySystem can be put into the container.  The system represents the physical relay device. Unfortunately you'll have to go from the protected switch -> function -> scheme -> system which is a bit clunky, but better than adding more unnecessary relationships.

Reply:
Change implemented and tested.